core/aop-runtime/test/fixtures/modules/hello_cross_cut/HelloCrossCut.ts

```
@@ -0,0 +1,69 @@
+  type: PointcutType.CLASS,
+  clazz: Hello,
+  methodName: 'hello',
+}, { adviceParams: crosscutAdviceParams })
+@Advice({
+  accessLevel: AccessLevel.PUBLIC,
+})
+export class CrosscutAdvice implements IAdvice<Hello, String> {
+  @Inject()
+  callTrace: CallTrace;
+
+  async beforeCall(ctx: AdviceContext<Hello, {}>): Promise<void> {
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)